base/ESMFL_Mod.F90

```diff
@@ -1262,9 +1261,8 @@ subroutine ESMFL_RegridStore (srcFLD, SRCgrid2D, dstFLD, DSTgrid2D, &
    !                           name="SRC 2D grid", rc=status)
    !_VERIFY(status)
    ! instead use the following...
-   pi = 4.0 * atan(1.0)
-   deltaX = 2.0*pi/gccpd(1)
-   deltaY = pi/(gccpd(2)-1) 
+   deltaX = 2.0*MAPL_PI/gccpd(1)
```

Comment:
@bena-nasa @atrayano  This seems like it would be non-zero-diff. Do you know who cares about π in ESMFL?

Reply:
Agreed, but ... if we are lucky, and they are zero diff, then we should make the switch.

Reply:
Based on this comment, seems this code was not tested in the run I performed since I got zero diff. Is there a suggestion as to what I can do to test if this change truly is zero diff?

Reply:
As I said at the meeting, I think we are safe. A grep of the full GEOSadas shows that no one actually seems to call `ESMFL_RegridStore`:
```
GMAO_Shared/MAPL_Base/ESMFL_Mod.F90
6: ESMFL_RegridStore
55:  public ESMFL_RegridStore ! only used for regridding using ESMF_FieldRegrid
1062:! !IROUTINE: ESMFL_RegridStore
1066:   subroutine ESMFL_RegridStore (srcFLD, SRCgrid2D, dstFLD, DSTgrid2D, &
1398:   end subroutine ESMFL_RegridStore
```

Now, maybe some odd one-off repo (a la AeroApps or GAAS or something) does, but my guess is they are far enough behind MAPL develop we can just blame any non-zero-diff on them moving up in time.